dependency/Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: retrieve test
+
+retrieve:
+	@cd retrieval; \
+	go build -o retrieve; \
+	./retrieve \
+	    --buildpack_toml_path=$(buildpackTomlPath) \
+		--output=$(output); \
+	rm retrieve
```

Comment:
```suggestion
	go run ./retrieve \
	    --buildpack_toml_path=$(buildpackTomlPath) \
	    --output=$(output)
```

Reply:
That's a good simplification. Will push commit after rebase.